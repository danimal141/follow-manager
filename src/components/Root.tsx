import * as React from 'react'
import { Text, View } from 'react-native'
import styles from './Root.style'

const Root = () => (
  <View style={styles.container}>
    <Text>Open up App.tsx to start working on your app!</Text>
  </View>
)

export default React.memo(Root)
