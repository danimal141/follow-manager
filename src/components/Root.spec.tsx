import * as React from 'react'
import * as renderer from 'react-test-renderer'

import Root from './Root'

describe('components/Root', () => {
  it('should be rendered successfully', () => {
    const component = renderer.create(<Root />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
