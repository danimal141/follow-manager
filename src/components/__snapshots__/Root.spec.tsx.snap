// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Root should be rendered successfully 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
</View>
`;
